src/main/java/org/continuouspoker/dealer/api/ManagementController.java

```diff
@@ -106,14 +112,36 @@ public void toggleRun(@PathParam(PARAM_GAME_ID) final long gameId) {
     }
 
     @GET
-    @Path("/{gameId}/log/{timestamp}")
+    @Path("/{gameId}/log/{timestamp}/limit={logLimit}")
     public List<LogEntry> getLogSince(@PathParam(PARAM_GAME_ID) final long gameId,
-            @PathParam("timestamp") final String timestamp) throws ObjectNotFoundException {
-        return service.getLogSince(gameId, timestamp);
+            @PathParam("timestamp") final String timestamp,
+            @PathParam("logLimit") final int logLimit) {
+        return service.getLogSince(gameId, timestamp, logLimit);
+    }
+
+    @GET
+    @Path("/{gameId}/log/limit={logLimit}")
+    public List<LogEntry> getLogByGameId(@PathParam(PARAM_GAME_ID) final long gameId,
+            @PathParam("logLimit") final int logLimit) {
+        return service.getLogByGameId(gameId, logLimit);
+    }
+
+    @GET
+    @Path("/{tournamentId}/log/limit={logLimit}")
+    public List<LogEntry> getLogByTournamentId(@PathParam(PARAM_TOURNAMENT_ID) final long tournamentId,
+            @PathParam("logLimit") final int logLimit) {
+        return service.getLogByTournamentId(tournamentId, logLimit);
+    }
+
+    @GET
+    @Path("/{roundId}/log/limit={logLimit}")
```

Comment:
I don't think this is how you specify query params correctly, please check again. 
@ QueryParam or @ RestQuery is more appropriate I think.